manifests/system.pp

```diff
@@ -44,9 +44,14 @@
     }
   }
 
+  $curl_binary = $::osfamily ? {
+    'FreeBSD' => '/usr/local/bin/curl',
+    default   => '/usr/bin/curl',
+  }
+
   exec { 'system-rvm':
-    path        => '/usr/bin:/usr/sbin:/bin',
-    command     => "/usr/bin/curl -fsSL https://get.rvm.io | bash -s -- --version ${actual_version}",
+    path        => '/usr/bin:/usr/sbin:/bin:/usr/local/bin',
+    command     => "${curl_binary} -fsSL https://get.rvm.io | bash -s -- --version ${actual_version}",
```

Comment:
can you change this so it just uses `curl -fsSL ...`
by using the correct path the command doesn't need to be qualified